client/a8c-for-agencies/sections/sites/needs-setup-sites/index.tsx

```diff
@@ -68,26 +65,23 @@ export default function NeedSetup( { licenseKey }: Props ) {
 
 	// Filter out sites that have a referral
 	const availableSites =
-		allAvailableSites.filter( ( { features }: NeedsSetupSite ) =>
-			isAutomatedReferralsEnabled ? ! features.wpcom_atomic.referral : true
+		allAvailableSites.filter(
+			( { features }: NeedsSetupSite ) => features.wpcom_atomic.referral
```

Comment:
I think this should be:

```suggestion
			( { features }: NeedsSetupSite ) => ! features.wpcom_atomic.referral
```

Reply:
Good catch! This has been fixed. 👍